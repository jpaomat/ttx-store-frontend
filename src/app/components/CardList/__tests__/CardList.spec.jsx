const Renderer = require('react-test-renderer/shallow');
const { CardList } = require('../CardList');

const renderer = new Renderer();

describe('Techtronix component', () => {
	it('should render component', () => {
		const component = renderer.render(<CardList />);
		expect(component).toMatchSnapshot();
	});
});