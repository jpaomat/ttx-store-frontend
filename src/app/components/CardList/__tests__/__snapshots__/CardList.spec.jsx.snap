// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Techtronix component should render component 1`] = `
<div
  className="card-container"
/>
`;
